lang/en.yml

```diff
@@ -33,6 +33,12 @@ en:
     CollapsePanel: 'Collapse panel'
     DELETED: Deleted.
     DropdownBatchActionsDefault: 'Choose an action...'
+    ErrorDetail: 'Error detail: {detail}'
+    ErrorMessage: '<p>Sorry, it seems there was a {errorcode} error.</p>'
```

Comment:
We should not have `<p>` tags in the translations - they should be directly in the template

Reply:
This is done _all over_ the place already.

Reply:
That's not a great reason :-)

All we need to do is change the template from `$Message` to `<p>$Message</p>`, unless I've missed something

Reply:
It was my argument for it not being worth the time it takes to change rather than for it being a good idea in the first place lol. Also not worth another round of ping pong though so I've moved the tag to the template.